src/main/asciidoc/micrometer/influx/main.adoc

```diff
@@ -0,0 +1,93 @@
+
+Grafana is open source metrics Dashboard platform.
+
+We will present the steps to prep, configure and rub Spring Cloud Data Flow's `Local` server.
+
+image::scdf-micrometer-influxdb-grafana-architecture.png[scaledwidth="100%"]
+
+==== Prerequisites
+
+* A Running Data Flow Shell
+include::{docs_dir}/shell.adoc[]
+* A running local Data Flow Server
+include::{docs_dir}/local-server.adoc[]
+* Running instance of link:http://kafka.apache.org/downloads.html[Kafka]
+* Spring Cloud Stream 2.x based https://github.com/spring-cloud-stream-app-starters/time/blob/master/spring-cloud-starter-stream-source-time/README.adoc[Time] and  https://github.com/spring-cloud-stream-app-starters/log/blob/master/spring-cloud-starter-stream-sink-log/README.adoc[Log] appcliations starters, pre-build with `io.micrometer:micrometer-registry-influx` dependency.
+
+==== Building and Running the Demo
+
+. Register the pre-build `time` and `log` applications
++
+```
+app register --name time2 --type source --uri file://<path-to-your-time-app-build>/time/apps/time-source-kafka/target/time-source-kafka-2.0.0.BUILD-SNAPSHOT.jar --metadata-uri file://<path-to-your-time-app-build>/time/apps/time-source-kafka/target/time-source-kafka-2.0.0.BUILD-SNAPSHOT-metadata.jar
+
+app register --name log2 --type sink --uri file://<path-to-your-log-app-build>/log/apps/log-sink-kafka/target/log-sink-kafka-2.0.0.BUILD-SNAPSHOT.jar --metadata-uri file://<path-to-your-log-app-build>/log/apps/log-sink-kafka/target/log-sink-kafka-2.0.0.BUILD-SNAPSHOT-metadata.jar
+```
++
+. Create InfluxDB and Grafana Docker containers
++
+```
+docker run -d --name grafana -p 3000:3000 grafana/grafana
+docker run -d --name influxdb -p 8086:8086 influxdb
+docker ps
+```
++
+You can connect to the InfluxDB and explore the measurements
++
+```
+docker exec -it <influx container-id> /bin/bash
+root:/# influx
+> show databases
+> use myinfluxdb
+> show measurements
+> select * from spring_integration_send limit 10
+```
++
+. Create and deploy the following stream
++
+```
+dataflow:> stream create --name t2 --definition "time2 | log2"
+
+dataflow:>stream deploy --name t2 --properties "app.*.management.endpoints.web.exposure.include=*,app.*.management.metrics.export.influx.db=myinfluxdb"
+```
+The `app.*.management.metrics.export.influx.db=myinfluxdb` instructs the `time2` and `log2` apps to use the `myifluxdb` database (created automatically).
++
+Then open in browser the link: http://localhost:8081/actuator/metrics and http://localhost:8082/actuator/metrics to verify that the metircs is 
++
+. Configure Grafana
++
+* Open Grafana UI (http://localhost:3000) and log-in (user: `admin`, password: `admin`).
+* Create InfluxDB datasource called: `influx_auto_DataFlowMetricsCollector` that  connects to our `myinfluxdb` influx database.
++
+image::grafana-influxdb-datasource.png[Add InfluxDB DataSource, scaledwidth="10%"]
++
+.DataSource Properties
```

Comment:
I have a slightly different screen, maybe an updated docker image since this PR was made.  Should we specify a specific label for the docker image?  More importantly, I get a warning `Network Error: Bad Gateway (502)` when configuring the data source.  I notice access is now listed as `Server (Default)` and not `direct`....  Prob the same thing.  I am able to query influx by doing `docker exec`... perhaps some setup on ubuntu 16.04?

Reply:
Good catch. I'll set the docker image tags and adjusted configurations.